tools/src/block-gen/Main.hs

```diff
@@ -0,0 +1,52 @@
+module Main where
+
+import           Universum
+
+import           Mockable            (runProduction)
+
+import qualified Data.HashMap.Strict as HM
+import qualified Data.Map            as M
+import           System.Directory    (doesDirectoryExist)
+
+import           Pos.Core            (StakeDistribution (..), addressHash,
+                                      genesisDevKeyPairs, makePubKeyAddress, mkCoin)
+import           Pos.Crypto          (toPublic, unsafeHash)
+import           Pos.DB              (closeNodeDBs, openNodeDBs)
+import           Pos.Generator.Block (AllSecrets (..), BlockGenParams (..), genBlocks)
+import           Pos.Genesis         (stakeDistribution)
+import           Pos.Txp.Core        (TxIn (..), TxOut (..), TxOutAux (..))
+import           Pos.Txp.Toil        (GenesisUtxo (..))
+
+import           Context             (initTBlockGenMode)
+import           Options             (BlockGenOptions (..), getBlockGenOptions)
+
+main :: IO ()
+main = do
+    BlockGenOptions{..} <- getBlockGenOptions
+    when bgoAppend $ checkExistence bgoPath
+    let keys = take (fromIntegral bgoNodesN) genesisDevKeyPairs
+    let dummyDistr = FlatStakes (fromIntegral bgoNodesN) (mkCoin $ fromIntegral bgoNodesN)
```

Comment:
It looks wrong to use development keys and genesis. This tool is mostly for production mode.